test/e2e/storage/persistent_volumes-local.go

```diff
@@ -265,6 +265,46 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				})
 			})
 
+			Context("Set fsGroup for local volume", func() {
+
+				It("should enforce fsGroup for one pod", func() {
+					fsGroup := int64(1234)
+					pod, err := createLocalPod(config, testVol, &fsGroup)
+					framework.ExpectNoError(err)
+					By("Checking fsGroup is enforced")
+					_, err = framework.LookForStringInPodExec(config.ns, pod.Name, []string{"ls", "-ld", volumeDir}, strconv.Itoa(int(fsGroup)), time.Second*10)
+					Expect(err).NotTo(HaveOccurred(), "failed: getting the right privileges in the file %v", int(fsGroup))
+				})
+
+				It("should not enforce fsGroup for second pod", func() {
+					if testVolType == DirectoryLocalVolumeType {
+						// TODO(cofyc): Test it when bug is fixed.
+						framework.Skipf("Skipped when volume type is %v", testVolType)
+					}
+					fsGroup1 := int64(1234)
```

Comment:
Can you create a helper method for creating a pod with fsgroup and verifying the fsgroup? It'll make this easier to read.

Reply:
Done.